internal/config/config.go

```diff
@@ -83,27 +43,32 @@ func LookupConfigValue(key string) (value string, ok bool) {
 	return os.LookupEnv(key)
 }
 
-func toWalgSettingName(waleSettingName string) string {
+func ToWalgSettingName(waleSettingName string) string {
 	return "WALG" + strings.TrimPrefix(waleSettingName, "WALE")
 }
 
 func GetSettingValue(key string) string {
 	if strings.HasPrefix(key, "WALE") {
-		walgKey := toWalgSettingName(key)
-		if val, ok := LookupConfigValue(walgKey); ok && len(val) > 0 {
+		walgKey := ToWalgSettingName(key)
+		if val, ok := LookupValue(walgKey); ok && len(val) > 0 {
 			return val
 		}
 	}
 
-	value, ok := LookupConfigValue(key)
+	value, ok := LookupValue(key)
 	if ok {
 		return value
 	}
 	return ""
 }
 
-func UpdateAllowedConfig(fields []string) {
+func UpdateAllowed(fields []string) {
 	for _, field := range fields {
 		allowedConfigKeys[field] = nil
 	}
 }
+
+func CheckAllowed(field string) bool {
```

Comment:
You use it only once. It doesn't desire to be separated function.

Reply:
Fixed.